.travis.yml

```diff
@@ -0,0 +1,32 @@
+dist: xenial
+
+language: go
+
+env:
+  - GO111MODULE=on
+
+go:
+  - 1.11.x
```

Comment:
super nitpick: according to the changelog the language spec in 1.11 and 1.12 are actually identical and there are just some improvements to the toolchain - idk if any of them are all that valuable but at the very least it _should_ be safe to bump this if you want

Reply:
I think I was having problems bumping it with `scrapedumper` -- this is copied over from that.  I can try moving it back again though and see if it works.